src/app/shell/workshop-details/side-menu/actions/actions.component.ts

```diff
@@ -1,18 +1,23 @@
+import { Store } from '@ngxs/store';
 import { Component, Input, OnInit } from '@angular/core';
+import { User } from 'src/app/shared/models/user.model';
 import { Workshop } from 'src/app/shared/models/workshop.model';
+import { RegistrationState } from 'src/app/shared/store/registration.state';
+import { Role } from 'src/app/shared/enum/role';
 
 @Component({
   selector: 'app-actions',
   templateUrl: './actions.component.html',
   styleUrls: ['./actions.component.scss']
 })
 export class ActionsComponent implements OnInit {
-
+  isDisplayed: boolean;
   @Input() workshop: Workshop;
 
-  constructor() { }
+  constructor(private store: Store) { }
 
   ngOnInit(): void {
+    this.isDisplayed = this.store.selectSnapshot<User>(RegistrationState.user).role !== Role.provider;
```

Comment:
I am wondering if this throws error for users that are not logged in yet

Reply:
already fixed